membership-attribute-service/conf/DEV.public.conf

```diff
@@ -31,7 +31,8 @@ authentication {
 
 mma.cors.allowedOrigins = [
   "https://profile.code.dev-theguardian.com",
-  "https://profile.thegulocal.com"
+  "https://profile.thegulocal.com",
```

Comment:
I think you'll need to make the corresponding change in the PROD config file

Reply:
Done, the PROD file has some code.dev urls in it. As subscriptions isn't really running a CODE env right now I've got mixed feelings about including them.

Reply:
Being the subs code environment url too.